vncdotool/rfb.py

```diff
@@ -181,7 +182,10 @@ def __init__(self):
         self._already_expecting = 0
         self._version = None
         self._version_server = None
-        self._zlib_stream = zlib.decompressobj(0)
+        try: # Automatically determine the window size from the zlib header. Only supported since zlib 1.2.3.5.
+            self._zlib_stream = zlib.decompressobj(0)
+        except builtins.ValueError:
```

Comment:
Can we simple catch ValueError rather than needing to import the builtin's namespace?

Reply:
I'll be unable to fix it until Jan the 11th if it's ok.

Reply:
no problem, appreciate the help